Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:latest
+
+ARG JUNO_WORKERS_NUMBER=1
+
+ENV JUNO_WORKERS_NUMBER=${JUNO_WORKERS_NUMBER}
+
+WORKDIR /app
+
+COPY go.mod go.sum ./
+
+RUN go mod download
```

Comment:
The image should copy contents and run `make install`. In other words, it should follow the same flow you would locally to build/install and run.

Reply:
updated [here](https://github.com/fissionlabsio/juno/pull/18/commits/ee3ec82212c9ae83914825072041047d3ffefec8)